File: autoescuela_rabel Consulta php/OpcionA.php
<?php

require("conexion.php");

 class OpcionA {

    private $name;
    private $isSelected;

    public function __construct($name) {
        $this->name = $name;
        $this->isSelected = false;
    }

    public function getName() {
        return $this->name;
    }

    public function setName($name) {
        $this->name = $name;
    }

    public function isSelected() {
        return $this->isSelected;
    }

    public function setSelected($selected) {
        $this->isSelected = $selected;
    }
	 
	 public function __toString()
{
    return self::class . ": " . $this->name . " " . $this->isSelected;
}

}

?>

File: Iscar ptoyecto andorid incluhe html de la web/app/src/main/java/com/example/iscar/Padres/OpcionA.java
package com.example.iscar.Padres;

import java.io.Serializable;

public class OpcionA implements Serializable {

    int idPregunta;
    private String name;
    private boolean isSelected;

    public OpcionA() {
    }

    public OpcionA(String name, boolean isSelected) {
        this.name = name;
        this.isSelected = isSelected;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public boolean isSelected() {
        return isSelected;
    }

    public void setSelected(boolean selected) {
        isSelected = selected;
    }

    public int getIdPregunta() {
        return idPregunta;
    }

    public void setIdPregunta(int idPregunta) {
        this.idPregunta = idPregunta;
    }

    @Override
    public String toString() {
        return "OpcionA{" +
                "name='" + name + '\'' +
                ", isSelected=" + isSelected +
                '}';
    }
}
